fragments/master.sh

```diff
@@ -25,6 +25,16 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
     exit 1
 fi
 
+RESTART_POLICY="no"
+START_MODE=$1
+
+if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
```

Comment:
in the current start_k8s() we launch kubelet as container. This is easy & nice for testing. But it is still not perfect for production. e.g.
- The manifest files are inside the hyperkube container and cannot be modified by customer. 
- The cloud provider has some issues if kubelet is executed as container

For production, it would be better if we match the setup of the salt scripts. In other words:
- kubelet is managed by systemd
- manifests are stored in /etc/kubernetes/manifests
- everthing works as documented on kubernetes.io

So, the start_k8s_poduction() would:

- docker run hyperkube install-on-host.sh    #installs everything on host
- kubectl start kubelet